content/en/roadmap.md

```diff
@@ -130,6 +132,7 @@ and make Flux watch for changes in ConfigMaps and Secrets referenced in Kustomiz
   - [x] [Add support for remote cluster authentication using Workload Identity](https://github.com/fluxcd/kustomize-controller/pull/1476)
   - [x] [Watch ConfigMaps and Secrets referenced in Kustomizations](https://github.com/fluxcd/flux2/issues/5446)
   - [x] [Extend the readiness evaluation of dependencies with CEL expressions](https://github.com/fluxcd/kustomize-controller/pull/1491)
+  - [x] [Fix multiple patch delete panic in strategic merge patches](https://github.com/fluxcd/kustomize-controller/issues/1306)
```

Comment:
No need to add bug fixes or pkg bumps to the roadmap. They will go in the changelog